aiohttp/http_writer.py

```diff
@@ -56,13 +59,18 @@ def _write(self, chunk):
             raise asyncio.CancelledError('Cannot write to closing transport')
         self._transport.write(chunk)
 
-    def write(self, chunk, *, drain=True, LIMIT=64*1024):
+    def write(self, chunk, *, drain=True, LIMIT=64 * 1024):
         """Writes chunk of data to a stream.
 
         write_eof() indicates end of stream.
         writer can't be used after write_eof() method being called.
         write() return drain future.
         """
+        self.on_chunk_sent.freeze()
+        self.loop.create_task(
```

Comment:
Spawning a new task is not what we need.
Technically `.write` is a coroutine, we need convert it to genuine async function.
Maybe I'll do it very soon in separate PR.

Reply:
Are you sure we don't want this in another task ? Whatever handler does asynchronously will slow down writing the response. 

Also `.write` isn't actually coroutine, because it's not called like:
```
                for chunk in self.body:
                    writer.write(chunk)
```
(without `await`) The couroutine returned is never triggered.
I guess I would have to track all the places there `StreamWriter.write` is used and add `await` in there. That's not undoable, but it's rather big refactor.

Reply:
Yes, I'm sure. Spawning a task without waiting for result has an ugly smell.
More important spawning a task may reorder received signals, e.g. user may get *request chunk* after *request finished*.

`.write` should be async method. I'm working on this refactoring (required for #2698 anyway).
I suggest waiting for my job done.

Reply:
Got it, I will wait until the refactoring is merged.